docs/resources/howto/wiki.md

```diff
@@ -0,0 +1,273 @@
+
+It can be created using
+
+```markdown
+1. First
+2. Second
+3. Third
+```
+
+#### Task list
+
+A task list looks like this.
+
+- [ ] First
+- [x] Second
+- [ ] Third
+
+It can be created using
+
+```markdown
+- [ ] First
+- [x] Second
+- [ ] Third
+```
+
+Note that ticks set by a user are not permanent and will be reset upon reloading the page.
+
+#### Tabbed list
+
+A tabbed list looks like this.
+
+=== "Entry 1"
+    Text 1
+=== "Entry 2"
+    Text 2
+=== "Entry 3"
+    Text 3
+
+It can be created using
+
+```markdown
+=== "Entry 1"
+    Text 1
+=== "Entry 2"
+    Text 2
+=== "Entry 3"
+    Text 3
+```
+
+### Text Boxes
+
+??? abstract "Paper Box"
+    Textbox for adding papers.
+    Code:
+
+    ```markdown
+    ??? abstract "Paper Box"
+        Textbox for adding papers.
+    ```
+
+??? info "Info Box"
+    Textbox for adding useful information.
+    Code:
+
+    ```markdown
+    ??? info "Info Box"
+        Textbox for adding useful information.
+    ```
+
+??? question "Question Box"
+    Textbox for answering commonly asked questions.
+    Code:
+
+    ```markdown
+    ??? question "Question Box"
+        Textbox for answering commonly asked questions.
+    ```
+
+??? tip "Tip Box"
+    Textbox for adding useful tips to novice users.
+    Code:
+
+    ```markdown
+    ??? tip "Tip Box"
+        Textbox for adding useful tips to novice users.
+    ```
+
+Note that appending a `+` to `???` will result in an expanded box by default.
+
+!!! note "Note Box"
+    Textbox for adding small notes.
+    Code:
+
+    ```markdown
+    !!! note "Note Box"
+        Textbox for adding small notes.
+    ```
+
+!!! warning "Warning Box"
+    Textbox for warning users of common pitfalls.
+    Code:
+
+    ```markdown
+    !!! warning "Warning Box"
+        Textbox for warning users of common pitfalls.
+    ```
+
+!!! danger "Danger Box"
+    Textbox for warning users of potential serious consequences if not executed properly.
+    Code:
+
+    ```markdown
+    !!! danger "Danger Box"
+        Textbox for warning users of potential serious consequences if not executed properly.
+    ```
+
+!!! todo "Todo Box"
+    Textbox for warning users that this webpage is still work in progress.
+    Code:
+
+    ```markdown
+    !!! todo "Todo Box"
+        Textbox for warning users that this webpage is still work in progress.
+    ```
+
+### Various other commands
+
+#### Tables
+
+Tables in markdown look like this.
+
+| Column 1      | Column 2    |
+| -----------   | ----------- |
+| Entry 11      | Entry 21    |
+| Entry 12      | Entry 22    |
+
+It can be created using
+
+```markdown
+| Column 1      | Column 2    |
+| -----------   | ----------- |
+| Entry 11      | Entry 21    |
+| Entry 12      | Entry 22    |
+```
+
+When creating a table, please ensure that column width is constant and that pipes (`|`) are aligned.
+
+#### Images
+
+To paste an image, use the following code.
+
+```markdown
+![Image](../../assets/images/tricks/placeholder.gif)
+```
+
```

Comment:
Centered images with caption are not possible in pure markdown, but can be easily achieved in `html`:

```html
<figure>
  <img src="../../../assets/images/something/image.png" width=90%>
  <figcaption>Figure: Something really amazing.</figcaption>
</figure>
```